edge/pkg/devicetwin/dtmanager/membership_test.go

```diff
@@ -87,8 +93,9 @@ func TestDealMembershipDetailInvalidMsg(t *testing.T) {
 	}
 
 	err := dealMembershipDetail(dtc, "t", m)
-	assert.Error(t, err)
-	assert.Equal(t, errors.New("assertion failed"), err)
+	if !reflect.DeepEqual(err, errors.New("assertion failed")) {
```

Comment:
```suggestion
	if err.Error()!="assertion failed" {
```
How about this?

Reply:
both are ok, but I think it's better to judge `error` itself, that's more accurate.

Reply:
/approve